deb-get

```diff
@@ -648,6 +648,12 @@ function init_repos() {
     if [ ! -e "${ETC_DIR}/01-main.repo" ]; then
         ${ELEVATE} tee "${ETC_DIR}/01-main.repo" <<<"${MAIN_REPO_URL}" >/dev/null
     fi
+    if [ ! -e "${ETC_DIR}/02-main.repo" ]; then
+         ${ELEVATE} sed 's/01/02/' "${ETC_DIR}/01-main.repo" > "${ETC_DIR}/02-main.repo"
```

Comment:
This initialises them as active
```suggestion
         ${ELEVATE} sed 's/01/02/' "${ETC_DIR}/01-main.repo" > "${ETC_DIR}/02-main.repo.disabled"
```

Reply:
But I think it best to _just_ move and give the user the option to turn off unwanted repos